assets/scripts/vendor.js

```diff
@@ -8,11 +8,16 @@ const btnMonthly = document.getElementById('monthly');
 const planValue = document.getElementById('plan-value');
 const totalValue = document.getElementById('total-value');
 const fee = document.getElementById('fee');
+const btnChange = document.getElementById('btn-change');
+const planType = document.getElementById('plan-type');
+const planPrice = document.getElementById('plan-price');
 const dollarSign = '$';
 const annualPrice = 59.99;
 const monthlyPrice = 66.65;
 const feeValue = 1.0;
 
+setCardPlanAndPrice('Annual Plan', dollarSign, annualPrice, '/year');
+
 function setTotalValue(dollar, billingCycle, feePrice) {
```

Comment:
In this function to have a better method semantic, you can use `currency` instead of `dollar` and `freeCost` instead of `feePrice`.

Reply:
Yes, indeed it is a better name. Thank you